### PR TITLE
Fix ws receive task

### PR DESCRIPTION
**Pull request by edenhaus**
# Proposed Changes

- [Fix ws receive task](https://github.com/home-assistant-libs/python-go2rtc-client/commit/dd00bca5b02ee2e01c8b06dffb6f265ff94af146)
- [Fix warning](https://github.com/home-assistant-libs/python-go2rtc-client/commit/7982666f9340f06c232f53ecae1beba447652dcf)
- [Increase required coverage to 97](https://github.com/home-assistant-libs/python-go2rtc-client/commit/ab4abd4a42c25e9c12a8a506a48d2e0cd0c067c5)

### CODE REVIEW

File: go2rtc_client/ws/client.py [hunk 1]
```diff
@@ -128,16 +134,8 @@ async def _receive_messages(self) -> None:
...
-            _LOGGER.exception("Unexpected error while receiving message")
-            raise
-        finally:
-            _LOGGER.debug(
-                "Websocket client connection from %s closed", self._server_url
-            )
-
-            if self.connected:
-                await self.close()
+            except Exception:  # pylint: disable=broad-except
```
Comment (bdraco) [inline go2rtc_client/ws/client.py:137]: This looks like it will swallow ConnectionReset exceptions which means this could be an infinite loop

Comment (bdraco) [reply go2rtc_client/ws/client.py:137]: Actually we swallow everything but `asyncio.CancelledError` and `asyncio.TimeoutError` upstream and convert it to `WSMessageError`

Comment (bdraco) [reply go2rtc_client/ws/client.py:137]: And `receive` has no timeout so nothing will get swallowed so this is fine.

Comment (bdraco) [inline go2rtc_client/ws/client.py:137]: In theory this should never happen

https://github.com/aio-libs/aiohttp/blob/5f61c7bff5bd0560a169b611f5c4b7e4b1ee3360/aiohttp/client_ws.py#L355

Comment (bdraco) [reply go2rtc_client/ws/client.py:137]: I guess its possible that `close()` gets called and then its already closing and https://github.com/aio-libs/aiohttp/blob/5f61c7bff5bd0560a169b611f5c4b7e4b1ee3360/aiohttp/_websocket/writer.py#L125 is hit

Comment (bdraco) [reply go2rtc_client/ws/client.py:137]: nope we swallow that as well https://github.com/aio-libs/aiohttp/blob/5f61c7bff5bd0560a169b611f5c4b7e4b1ee3360/aiohttp/client_ws.py#L278

Comment (bdraco) [reply go2rtc_client/ws/client.py:137]: So it _should_ never raise here

**Review by bdraco — APPROVED** [review]
`self._client.receive()` should only ever raise `asyncio.CancelledError` or `asyncio.TimeoutError` (only if a timeout is specified) so the try/except is likely unneeded but it probably isn't hurting anything unless this code is performance sensitive.

**Review by bdraco — APPROVED** [review]

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**